demo/demo3.html

```diff
@@ -52,11 +53,11 @@
 
   <app-drawer-layout fullbleed>
 
-    <div drawer>
+    <app-drawer>
       <app-toolbar>App name</app-toolbar>
-    </div>
+    </app-drawer>
 
-    <app-header-layout main has-scrolling-region>
+    <app-header-layout has-scrolling-region>
 
       <app-header condenses reveals effects="waterfall">
         <app-toolbar>
```

Comment:
Ah, tried to remove some CSS from app-drawer-layout but removed too much. Added back support for the `[fullbleed]` attribute and relative positioning/100% height for its content container.